### PR TITLE
Ft user logout

### PR DESCRIPTION
**Pull request by Azutech**
#ft/user logout, Implement user logout
PR Description Template
 #### Description #### Type of change
 Please select the relevant option - [ ] Bug fix (non-breaking changes which fixes an issue)
 - [X] New feature (non-breaking changes which adds functionality)
 - [ ] Breaking change (fix or feature that could cause existing functionality to not work as expected)
 - [ ] This change requires a documentation update
 - [X] Chore #### How Has This Been Tested?
 please describe the test that you ran to verify your changes 
 - [ ] End to end
 - [ ] Integration #### Checklist: - [X] My code follows the style guidelines of this project
 - [X] I have linted my code prior to submission
 - [ ] I have made corresponding changes to the documentation
 - [X] My changes generates no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Azutech** [comment]
Logout feature has been implemented

File: src/services/Auth/index.ts [hunk 1]
```diff
@@ -78,3 +82,60 @@ export const loginService = async (req: Request, res: Response) => {
     res.cookie("handout_token", token);
     res.status(200).json({ message: "Login Succesful", token });
 };
+
+export const forgotPasswordService = async (req: Request, res: Response) => {
+
+  const code = digitalCode()
+  console.log(code)
```
Comment (forestbaba) [inline src/services/Auth/index.ts]: Take out this

File: src/services/Auth/index.ts [hunk 2]
```diff
@@ -78,3 +82,60 @@ export const loginService = async (req: Request, res: Response) => {
...
+    try {
+      const result = joi.object({email: joi.string().email().required()})
+      const { error } = result.validate(req.body)
+      if (error)
+      return res.json({
+        status: 'error',
+        error: 'Enter a valid email'
+      })
+
+      const user = await User.findOne({email: req.body.email})
```
Comment (forestbaba) [inline src/services/Auth/index.ts]: Why are you calling a model?

File: src/services/Auth/index.ts [hunk 3]
```diff
@@ -78,3 +82,60 @@ export const loginService = async (req: Request, res: Response) => {
...
+      if(!user)
+      return res.json({
+        status: "error",
+        error: "This email does not exist"
+      })
+  
+      let token = await tokens.findOne({userId: user.id});
```
Comment (forestbaba) [inline src/services/Auth/index.ts]: Why are you calling a model?

**Review by forestbaba — COMMENTED** [review]
IS this code a branch off of dev??

File: src/services/Auth/index.ts (hunk 1 above)
Comment (Azutech) [reply src/services/Auth/index.ts]: I will resolve it now

File: src/services/Auth/index.ts (hunk 3 above)
Comment (Azutech) [reply src/services/Auth/index.ts]: it has been updated

File: src/services/Auth/index.ts (hunk 2 above)
Comment (forestbaba) [reply src/services/Auth/index.ts]: This hasn't been addressed yet

Comment (Azutech) [reply src/services/Auth/index.ts]: Yes it has

**Pull request closed without merging**